rpc/testservice_test.go

```diff
@@ -93,8 +93,8 @@ func (s *testService) Rets() (string, error) {
 }
 
 //lint:ignore ST1008 returns error first on purpose.
-func (s *testService) InvalidRets1() (string, error) {
-	return "", nil
+func (s *testService) InvalidRets1() (error, string) {
```

Comment:
ignore the ST1008 check here , and then it won't include in the callbacks

Reply:
InvalidRets1 is made as wrong format method for testing

Reply:
So we expect it is wrong